### ISSUE TITLE
Pooling is not working

### ISSUE THREAD

**Issue by mareksugar** [proddy/EMS-ESP-Boiler#6]
I got problem with receiving messages from pooling boiler.
It is possible to show in log all message - including boiler poll request ?

**Comment by proddy** [comment]
if you set logging to verbose you'll see all messages on the EMS bus.

The poll requests are sent very quickly and you'll get a lot of data if you want to capture them. You could add a line after #342 in ems.cpp like

```c
myDebug("received poll 0x%02x", value);
```

**Comment by mareksugar** [comment]
so is full of poll requests:
0x09received poll 0x8breceived poll 0x89received poll 0x09received poll 0x89received poll 0x09received poll 0x89received poll 0x09received poll 0x8creceived poll 0x89received poll 0x09received poll 0x8dreceived poll 0x89received poll 0x09received poll 0x89received poll 0x09received poll 0x89received poll 0x09received poll 0x89received poll 0x09received poll 0x89received poll 0x09received poll 0x89received poll 0x09received poll 0x89received poll 0x09received poll 0x8ereceived poll 0x89received poll 0x09received poll 0x8freceived poll 0x89received poll 0x09received poll 0x89received poll 0x09received poll 0x89received poll..

what are 0b 89 messages?
should it be 0b 88?

![image](https://user-images.githubusercontent.com/33668626/48211629-2b7e1a80-e37a-11e8-9ce8-b4b9c062c99d.png)

**Comment by proddy** [comment]
this looks fine. It means on the ems bus you have a device with id 0x09. I'm not sure what it is. The boiler (master) sends out a poll with a single byte 0x89 and the 0x09 device responds back with sending its own echo. They are only single bytes, not messages.

What are you trying to do exactly?

**Comment by proddy** [comment]
it should work without the poll. It's actually disabled by default. I see there's a lot of noise on the line too which could be preventing the sending/Tx from working, which could be caused if you're using a different circuit that doesn't handle a steady ESP8266.

**Comment by proddy** [comment]
looking at https://emswiki.thefischer.net/doku.php?id=wiki:ems:ems-telegramme and https://www.buderus.dk/files/200708172223530.Buderus%20Logamatic%20EMS.pdf it does seem the EM10 sits on the EMS bus with an ID 0x09. Worth checking.

**Comment by mareksugar** [comment]
but now I dont have EM10 connected right now
ok BC10 has address 09. it is front display, I have no idea that front display has address too
http://img0064.popscreencdn.com/104384294_buderus-regelung-logamatic-mc10-mit-rc35-brm10-u-bc10-.jpg

**Comment by proddy** [comment]
Ok, so Polling is working and I'm not sure what the actual problem is - most probably the circuit. I'm closing this issue and please open a new one if you need help with another issue.
